inference_server/utils/utils.py

```diff
@@ -96,15 +87,6 @@ def print_rank_n(*values, rank: int = 0) -> None:
         print(*values)
 
 
-def get_dtype_from_model_name(model_name: str) -> str:
-    if model_name == BIGSCIENCE_BLOOM:
-        return "bf16"
-    elif model_name == DS_INFERENCE_BLOOM_FP16:
-        return "fp16"
-    elif model_name == DS_INFERENCE_BLOOM_INT8:
-        return "int8"
-
-
 def get_torch_dtype(dtype_str: str) -> torch.dtype:
     if dtype_str == "bf16":
```

Comment:
nit: The type hints in the `get_str_dtype` func below are the wrong way around. It should take a `torch.dtype` and return a `str`